apps/galaxy/src/layouts/HeaderLayout.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from "react";
+import Head from "next/head";
+
+interface HeaderLayoutProps {
+  children: ReactNode;
+  title?: string;
+}
+
+// pass title value to set the head title， e.g. In 404 page, write <HeaderLayout title="404"></HeaderLayout>
+const HeaderLayout: React.FC<HeaderLayoutProps> = ({ children, title }) => {
```

Comment:
We might need to rename this, `HeaderLayout` doesn't really represent the purpose of this component as its sole responsibility is to set a page title for us.

Suggest we use `<Head>`?

Reply:
because `Head` is already imported from `next/head`, so I change it to PageTitle